From 9140ffcdae2a71ced6e75670c11b06f1f4609eba Mon Sep 17 00:00:00 2001
From: Emmanuele Bassi <ebassi@gnome.org>
Date: Mon, 1 May 2017 15:12:06 +0100
Subject: [PATCH 1/1] Remove the g_auto* macros for Gegl

Gegl 0.3.15 provides the definitions for using GeglOperation and other
types with the g_auto* macros, now that bug 780083 has been fixed.

https://bugzilla.gnome.org/show_bug.cgi?id=782012
---
 configure.ac      |  2 +-
 src/photos-gegl.h | 14 --------------
 2 files changed, 1 insertion(+), 15 deletions(-)

diff --git a/configure.ac b/configure.ac
index 486263a..68396c5 100644
--- a/configure.ac
+++ b/configure.ac
@@ -38,7 +38,7 @@ GLIB_GSETTINGS
 
 CAIRO_MIN_VERSION=1.14.0
 GDATA_MIN_VERSION=0.15.2
-GEGL_MIN_VERSION=0.3.14
+GEGL_MIN_VERSION=0.3.15
 GFBGRAPH_MIN_VERSION=0.2.1
 GLIB_MIN_VERSION=2.44.0
 GOA_MIN_VERSION=3.8.0
diff --git a/src/photos-gegl.h b/src/photos-gegl.h
index bb6f4d8..11d0176 100644
--- a/src/photos-gegl.h
+++ b/src/photos-gegl.h
@@ -60,20 +60,6 @@ gboolean         photos_gegl_processor_process_finish     (GeglProcessor *proces
 
 void             photos_gegl_remove_children_from_node    (GeglNode *node);
 
-G_DEFINE_AUTOPTR_CLEANUP_FUNC (GeglOperation, g_object_unref);
-
-typedef struct _GeglOperationMeta GeglOperationMeta;
-G_DEFINE_AUTOPTR_CLEANUP_FUNC (GeglOperationMeta, g_object_unref);
-
-typedef struct _GeglOperationPointFilter GeglOperationPointFilter;
-G_DEFINE_AUTOPTR_CLEANUP_FUNC (GeglOperationPointFilter, g_object_unref);
-
-typedef struct _GeglOperationPointRender GeglOperationPointRender;
-G_DEFINE_AUTOPTR_CLEANUP_FUNC (GeglOperationPointRender, g_object_unref);
-
-typedef struct _GeglOperationSink GeglOperationSink;
-G_DEFINE_AUTOPTR_CLEANUP_FUNC (GeglOperationSink, g_object_unref);
-
 G_END_DECLS
 
 #endif /* PHOTOS_GEGL_H */
-- 
2.12.2

